src/run.php

```diff
@@ -42,7 +41,11 @@
     $app = new HiveApplication($config, $logger, $inputState, $dataFolder);
 
     if ($app['action'] !== 'run') {
-        $app['logger']->setHandlers(array(new NullHandler(Logger::INFO)));
+        // On sync actions -> log only errors
+        $logger->setHandlers([
```

Comment:
proč tam nemůžou být defaultní handlery?

Reply:
@ondrajodas v principe mozu ... chcel som to spravit ako v `php-component`, ze sa loguju iba chyby, ale napr `INFO` nie ... kedze pri `sync action` nemozes nic vypisat na `stderr/stdout` pokial sa nevyskytla chyba,`exit code !=0` .... lebo na vystupe sa ocakava JSON napr. `("success": true }`, ... ale mozem to vyhodit, mam to tak upravit?